driver/src/price_finding/linear_optimization_price_finder.rs

```diff
@@ -291,14 +294,18 @@ pub mod tests {
         });
 
         let expected_solution = models::Solution {
-            objective_value: U256::from_dec_str("15854632034944469292777429010439194350").ok(),
             prices: vec![14_024_052_566_155_238_000, 1_526_784_674_855_762_300],
             executed_sell_amounts: vec![0, 318_390_084_925_498_118_944],
             executed_buy_amounts: vec![0, 95_042_777_139_162_480_000],
         };
 
         let solution = deserialize_result(&json, 2).expect("Should not fail to parse");
         assert_eq!(solution, expected_solution);
+
+        // TODO(nlordell): we need orders in order to calcaulate this utility
+        let expected_objective_value =
+            U256::from_dec_str("15854632034944469292777429010439194350").ok();
+        let _ = expected_objective_value;
```

Comment:
Would this be a place to use some sort of mock?

Reply:
I think here we would update the `expected_objective_value` to be what is actually expected given the orders, solution and the new objective function and then test against that. I left a TODO so I don't forget to come back and fix this test.